go/vt/vtgate/planner.go

```diff
@@ -43,10 +52,53 @@ func NewPlanner(vschema *vindexes.VSchema, cacheSize int) *Planner {
 	return plr
 }
 
+// LoadVSchema loads the VSchema from the topo. The function does
+// not return an error. It instead logs warnings on failure.
+func (plr *Planner) LoadVSchema() {
+	formal := &vindexes.VSchemaFormal{
+		Keyspaces: make(map[string]vindexes.KeyspaceFormal),
+	}
+	keyspaces, err := plr.serv.GetSrvKeyspaceNames(context.TODO(), plr.cell)
```

Comment:
Usually you would pass in the Context into this function, let the higher levels create the one it's gonna use (that ends up being main() somewhere eventually, and then it's ok)